### ISSUE TITLE
Pool members aren't being added to AS3 pool

### ISSUE THREAD

**Issue by raZorTT** [f5devcentral/f5-as3-config-converter#37]
### Environment
 * BIG-IP Version: 12.1.5.3

### Summary
Tried using vscode-f5-chariot (v1.12.0) to convert TMSH config into AS3. When I run the conversion over a simple virtual server with a single pool member, the member isn't part of the AS3 output.

### Steps To Reproduce
1. Convert following config:
```conf
ltm virtual /Common/vs_hcf_443 {
    destination /Common/192.168.0.1:443
    ip-protocol tcp
    mask 255.255.255.255
    pool /Common/hcf_80
    profiles {
        /Common/tcp { }
    }
    source 0.0.0.0/0
    translate-address enabled
    translate-port enabled
}
ltm pool /Common/hcf_80 {
    description hcf_80
    members {
        /Common/10.0.0.1:80 {
            address 10.0.0.1
        }
    }
    monitor /Common/http
}
ltm node /Common/10.0.0.1 {
    address 10.0.0.1
}
```

2. Observe the AS3 output:
```out
{
    "class": "ADC",
    "schemaVersion": "3.27.0",
    "id": "urn:uuid:d58e0962-5bce-4403-9bcf-8c863a24a181",
    "label": "Converted Declaration",
    "remark": "Auto-generated by AS3 Config Converter",
    "Common": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared",
            "vs_hcf_443": {
                "layer4": "tcp",
                "pool": "hcf_80",
                "translateServerAddress": true,
                "translateServerPort": true,
                "class": "Service_Generic",
                "virtualAddresses": [
                    "192.168.0.1"
                ],
                "virtualPort": 443,
                "persistenceMethods": [],
                "snat": "none"
            },
            "hcf_80": {
                "remark": "hcf_80",
                "members": [
                    {
                        "addressDiscovery": "static",
                        "servicePort": null
                    }
                ],
                "monitors": [
                    {
                        "bigip": "/Common/http"
                    }
                ],
                "class": "Pool"
            }
        }
    }
}
```
3. Observe the ACC log output
```log
[2021-06-03T05:40:21.412Z] [INFO]: ACC METADATA {
  recognized: {
    'ltm virtual /Common/vs_hcf_443': {
      destination: '/Common/192.168.0.1:443',
      'ip-protocol': 'tcp',
      mask: '255.255.255.255',
      pool: '/Common/hcf_80',
      profiles: '{',
      '/Common/tcp': {},
      '}': '',
      source: '0.0.0.0/0',
      'translate-address': 'enabled',
      'translate-port': 'enabled'
    },
    'ltm pool /Common/hcf_80': {
      description: 'hcf_80',
      members: '{',
      '}': '',
      monitor: '/Common/http'
    },
    'ltm node /Common/10.0.0.1': { address: '10.0.0.1' }
  },
  supported: {
    'ltm virtual /Common/vs_hcf_443': {
      destination: '/Common/192.168.0.1:443',
      'ip-protocol': 'tcp',
      mask: '255.255.255.255',
      pool: '/Common/hcf_80',
      profiles: '{',
      '/Common/tcp': {},
      '}': '',
      source: '0.0.0.0/0',
      'translate-address': 'enabled',
      'translate-port': 'enabled'
    },
    'ltm pool /Common/hcf_80': {
      description: 'hcf_80',
      members: '{',
      '}': '',
      monitor: '/Common/http'
    }
  },
  unSupported: { 'ltm node /Common/10.0.0.1': { address: '10.0.0.1' } },
  declarationInfo: {
    classes: { Pool: 1, Service_Generic: 1 },
    maps: {
      applications: [ '/Common/Shared' ],
      objects: [ '/Common/Shared/vs_hcf_443', '/Common/Shared/hcf_80' ],
      tenants: [ '/Common' ]
    },
    total: 2
  }
}
```
### Expected Behavior
I am expecting to see the member (10.0.0.1) as part of the AS3 pool output.

### Actual Behavior
As you can see from the output there is no pool member being output. Looking at the ACC log it seems to see the pool, but it doesn't output the member IP then either?

```
'ltm pool /Common/hcf_80': {
      description: 'hcf_80',
      members: '{',
      '}': '',
      monitor: '/Common/http'
    }
```

**Comment by mdditt2000** [comment]
Added fine testing f5-chariot using mac

```
            "hcf_80": {
                "remark": "hcf_80",
                "members": [
                    {
                        "addressDiscovery": "static",
                        "servicePort": 80,
                        "serverAddresses": [
                            "10.0.0.1"
                        ],
                        "shareNodes": true
                    }
                ],
                "monitors": [
                    {
                        "bigip": "/Common/http"
                    }
                ],
                "class": "Pool"
            }
```
https://github.com/mdditt2000/f5-appsvcs-acc/tree/master/Github/37

Trying Windows

**Comment by mdditt2000** [comment]
breaks using windows. Mac is fine https://github.com/mdditt2000/f5-appsvcs-acc/tree/master/Github/37

**Comment by mdditt2000** [comment]
Create Jira CHARON-321 for PM tracking

**Comment by DumpySquare** [comment]
I think this is related to #28.  I will dig into that one and report back.

**Comment by DumpySquare** [comment]
This should be fixed with the release of vscode-f5-chariot v1.13.0 released 7.6.2021.  There was a problem with line returns in windows.  The fix was to standardize all line returns on execution.

Please confirm this is working with latest version.

**Comment by mdditt2000** [comment]
Change By: | @v-zhuravlev-f5 
-- | --
Done
In Code Review Done

Closing issue
